Cargo.toml

```diff
@@ -1,8 +1,7 @@
 [package]
-
 name = "openstack"
 description = "OpenStack SDK for Rust"
-version = "0.4.2"  # remember to update html_root_url
+version = "0.5.0"  # remember to update html_root_url
```

Comment:
Please do update html_root_url (it's in lib.rs)

Reply:
Thanks for the reminder, it is now done.